pkg/steps/input_image_tag.go

```diff
@@ -72,14 +72,21 @@ func (s *inputImageTagStep) Run(ctx context.Context) error {
 }
 
 func (s *inputImageTagStep) run(ctx context.Context) error {
-	logrus.Infof("Tagging %s into %s:%s.", s.config.BaseImage.ISTagName(), api.PipelineImageStream, s.config.To)
-
+	var from *coreapi.ObjectReference
```

Comment:
Wouldn't it be better moving this whole block of new code right after this check?
```golang
if _, err := s.Inputs(); err != nil {
...
}
```